rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -406,26 +312,37 @@ async def create_new_template(
         for link in graph.links:
             link.source_id = id_map[link.source_id]
             link.sink_id = id_map[link.sink_id]
-
-        return await autogpt_server.data.graph.create_graph(graph, is_template=True)
+        updated_graph = await autogpt_server.data.graph.create_graph(graph)
+        await autogpt_server.data.graph.deactivate_graph(
+            graph_id=graph.graph_id, version=graph.version
```

Comment:
`graph.version` has just been incremented, so this will deactivate the newly created graph.